src/components/Settings.svelte

```diff
@@ -654,6 +655,18 @@
 				}
 			}}
 		/>
+		<span class="label-text col-span-2">Max lines</span>
+		<input
+			type="number"
+			class="input input-bordered h-8 mb-2 col-span-2"
+			min="0"
+			bind:value={$maxLines$}
+			on:blur={() => {
```

Comment:
it would probably be nice/consistent with other settings (like duplicate preventation) if there would be a confirm dialog for adjusting the current list of elements to max of n if user wants to do so